node/octicons/index.js

```diff
@@ -0,0 +1,60 @@
+const icons = require('./build/data.json')
+
+for (const key of Object.keys(icons)) {
+  const icon = icons[key]
+
+  // Set the symbol for easy access
+  icon.symbol = key
+
+  // Set all the default options
+  icon.options = {
+    version: '1.1',
```

Comment:
Is the `version` attribute necessary? Looks like all browsers just ignore it: https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed